src/idiom284ex.fz

```diff
@@ -0,0 +1,6 @@
+ex284 is
+
+  n := 5
+  a :=  (0 : nil).cycle.take n
```

Comment:
I'd prefer `[0].cycle.take n` here. Or `array n _->0`

Reply:
`array n _->0` is what I used first, but from Fridtjof's comment above, I understood that it is not lazy. It doesn't matter in this specific example, because it is printed right afterwards, but I think it is still good to show the lazy variant.
Is `[0].cycle.take n` lazy?

Reply:
> Is [0].cycle.take n lazy?
Yes